example/basic/main.go

```diff
@@ -65,21 +64,25 @@ func main() {
 
 	defer exporter.Close()
 
-	processor := basic.New(
-		simple.NewWithExactDistribution(),
-		exporter,
+	c := controller.New(
+		processor.NewFactory(
+			selector.NewWithHistogramDistribution(
+				histogram.WithExplicitBoundaries([]float64{1.0, 2.0, 4.0, 8.0}),
+			),
+			exporter,
+			processor.WithMemory(true),
+		),
```

Comment:
That didn't work for me. Metrics weren't actually exported. Had to add this to get it actually exporting, otherwise the exporter was always `nil`
```suggestion
		),
		controller.WithExporter(exporter),
		controller.WithCollectPeriod(time.Second*10),
```

Reply:
Yep, same here. :slightly_smiling_face: 
The `controller.WithCollectPeriod(time.Second*10)` is optional as `collect()` is called down below, but I think it would still be nice to include it here as users might copy-paste from this example.

Reply:
I think it is also the default. The infinite loop that I proposed below does not call collect explicitly, and I thought it might be nice to show how to set up a different export interval in the example, because that is something users might want to do.